packages/components/src/components/install-guide/record-instructions/JavaScript.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <section>
+    <p>
+      When you run your JavaScript code with the
+      <code class="inline">npx appmap-node</code> command, AppMap will be enabled for recording. For
+      example, if you normally run your code with
+      <v-code-snippet clipboard-text="yarn start" :kind="codeSnippetType" />
+      simply run with
+      <v-code-snippet clipboard-text="npx appmap-node yarn start" :kind="codeSnippetType" />
+      to enable recording.
+    </p>
+    <h2>Choose the best recording method for this project</h2>
+    <br />
+    <template v-if="testFramework">
+      <section class="recording-method">
+        <h3>
+          <i class="header-icon"><TestsIcon /></i>Tests recording<span class="recommended-badge"
```

Comment:
For Navie, which is an important focus, Requests Recording is often the best way to get started. So, I'd like drop the Recommended Badge from Tests recording. If Tests Recording is the only option (due to no Express detected) then it will be de facto recommended.

Reply:
I'm going to remove the badge and move the test recording section to the bottom.